pyaccel/optics/beam_envelope.py

```diff
@@ -552,8 +552,9 @@ def calc_beamenvelope(
     envelopes += envelopes.transpose(0, 2, 1)
     envelopes /= 2
 
-    accelerator.radiation_on = rad_stt
-    accelerator.cavity_on = cav_stt
+    if fixed_point is None:
```

Comment:
@jquentino , the var `fixed_point` is being overwritten when `None` is passed (line 508). You can not rely on it anymore to restore the original values of `radiation_on` and `cavity_on` flags.

Reply:
Well noted @xresende. I created a flag to register if the fixed point is passed or not.